### ISSUE TITLE
2016 Layout needs some adjustments in the watch page

### ISSUE THREAD

**Issue by UnderscoreAngel** [lightbeam24/CustomTube#5]
![imagen](https://github.com/lightbeam24/customtube/assets/105066360/c6f11267-2a5d-4b0b-af62-7fbfcf729afe)

**Comment by lightbeam24** [comment]
I don't know how to fix that properly yet. For some reason, **whether or not this issue happens depends on your zoom level.** This issue doesn't happen for me on 100% zoom (I'm using a 1080p monitor), however it does happen when I zoom in or out. It could also depend on the OS you're using (because of font rendering differences), but that's just theoretical (CustomTube was only tested on Linux).

Basically, this would have to be fixed for EVERY single zoom level, and that just isn't feasible. There are just too many factors that play into this tiny issue.

I do have an idea that would theoretically fix this, but I'm not entirely sure, and it's very, very low on the priority list.

**Comment by UnderscoreAngel** [comment]
> I don't know how to fix that properly yet. For some reason, **whether or not this issue happens depends on your zoom level.** This issue doesn't happen for me on 100% zoom (I'm using a 1080p monitor), however it does happen when I zoom in or out. It could also depend on the OS you're using (because of font rendering differences), but that's just theoretical (CustomTube was only tested on Linux).
> 
> Basically, this would have to be fixed for EVERY single zoom level, and that just isn't feasible. There are just too many factors that play into this tiny issue.
> 
> I do have an idea that would theoretically fix this, but I'm not entirely sure, and it's very, very low on the priority list.

yes this happens on 7ktTube aswell, i guess the only real way to fix it is to make different adjustments for different resolutions, and it's way too complicated, i use 80% zoom, but other than that i have no idea either
